.github/workflows/test-e2e-cli.yml

```diff
@@ -98,4 +58,6 @@ jobs:
         shell: bash
         env:
           VERSION: ${{ inputs.version }}
-          TEST_KBC_PROJECTS: ${{ env[matrix.projects] }}
+          TEST_KBC_PROJECTS_LOCK_HOST: ${{ secrets.TEST_KBC_PROJECTS_LOCK_HOST }}
```

Comment:
Host should be variable not secret, now, I dont know which Redis is CI using (sorry, I didn't mentioned it in the previous PR).

Reply:
Will rework both.